Documentation/test-suites-overview.md

```diff
@@ -0,0 +1,240 @@
+static and runtime checks.
+
+The list of tested platforms can be found in [meta-mender
+repository](https://github.com/mendersoftware/meta-mender/tree/master/tests/build-conf).
+A "platform", in this context is a combination of CPU architecture (x86, ARM),
+boot loader integration (UEFI Grub, u-boot, bios) and a file system (ext4,
+ubifs). Following is a list of the current platforms under test:
+
+* qemux86-64-uefi-grub
+* vexpress-qemu
+* vexpress-qemu-flash
+* raspberrypi3
+* raspberrypi4
+* beagleboneblack
+* qemux86-64-bios-grub-gpt
+* qemux86-64-bios-grub
+* vexpress-qemu-uboot-uefi-grub  
+
+### Running locally
+
+A full Yocto environment (and at least one previously build image) is required
+to execute the tests. A complete recipe of how we build and test this on CI can
+be found at [mender-qa
+repository](https://github.com/mendersoftware/mender-qa/blob/master/scripts/yocto-build-and-test.sh).
+
+The following recipe gives an example of how to build and test the
+`qemux86-64-uefi-grub` platform.
+
+```
+# Initialize a new Yocto build environment
+source oe-init-build-env build-qemux86-64-uefi-grub
+# Add meta-mender build layers
+bitbake-layers add-layer ../meta-mender/meta-mender-core
+bitbake-layers add-layer ../meta-mender/meta-mender-demo
+bitbake-layers add-layer ../meta-mender/meta-mender-qemu
+# Add meta-mender test layer
+bitbake-layers add-layer ../meta-mender/tests/meta-mender-ci
+# Take the default Yocto configuration for this platform
+cp ../meta-mender/tests/build-conf/qemux86-64-uefi-grub/local.conf conf/
+# Set MENDER_ARTIFACT_NAME
+cat >> conf/local.conf <<EOF
+MENDER_ARTIFACT_NAME = "mender-image-local"
+EOF
+# Build full image
+bitbake core-image-full-cmdline
+# Run the tests!
+cd ../meta-mender/tests/acceptance/
+python3 -m pytest -p no:xdist --bitbake-image core-image-full-cmdline --board-type=qemux86-64
+```
+
+## Backend tests
+
+Test suite       | backend integration tests
+---------------- | ----------
+ISTQB equivalent | System integration testing and server-only Acceptance testing
+Test object      | The Mender backend server, as a whole
+Source code      | [integration/backend-tests](https://github.com/mendersoftware/integration/tree/master/backend-tests)
+How to run       | [`./run`](https://github.com/mendersoftware/integration/blob/master/backend-tests/run)
+Dependencies     | Docker, docker-compose
+
+This test suite verifies the interfaces and behavior of a stand-alone Mender
+server installation as a whole. A real client is not used, and instead API calls
+to the different endpoints simulates Mender clients connecting to the Mender
+server.
+
+Refer to the
+[README](https://github.com/mendersoftware/integration/tree/master/backend-tests)
+for more details of this test suite.
+
+
+## Integration tests
+
+Test suite       | integration tests
+---------------- | ----------
+ISTQB equivalent | System testing and Acceptance testing
+Test object      | The Mender product, as a whole
```

Comment:
this does not include any UI testing at all...

Reply:
Right, good point I'll rephrase this.